backend/controllers/event_controllers/admin_event_controller.py

```
@@ -0,0 +1,618 @@
+    if old_event is None:
+        return {"success": False, "msg":"Error: Original event not found!"}
+    
+    success = await update_event(event_id, event)
+    if not success['success']:
+        return success
+    
+    try:
+        recalculate = await recalculate_published_dates(event_id)
+        if not recalculate['success']:
+            return {'success':False, 'msg':f"Warning! Event update was successful, but we failed to recalculate the dates for event instances! Error: {recalculate['message']}"}
+    except Exception as e:
```

Ruff (0.14.5)

Do not catch blind exception: `Exception`

(BLE001)